github/repos_hooks_test.go

```diff
@@ -296,3 +296,261 @@ func TestRepositoriesService_TestHook_invalidOwner(t *testing.T) {
 	_, err := client.Repositories.TestHook(ctx, "%", "%", 1)
 	testURLParseError(t, err)
 }
+func TestBranchWebHookPayload_Marshal(t *testing.T) {
```

Comment:
Please add blank line between tests (before this line).

Reply:
added thanks